README_maintainers.md

```diff
@@ -16,7 +16,12 @@ Main sections:
   - [Linux Core Dumps](#linux-core-dumps)
   - [Windows Core Dumps](#windows-core-dumps)
 - [Unittests](#unittests)
-
+  - [invoking driver tests](#driver-tests) via scripts/unittests
+  - [capturing test communication](#running-tcpdump--windump-for-the-sut) but
```

Comment:
I would rephrase this as "capturing test HTTP communication".
Additionally, is the "--" in the anchor intentional?

Reply:
`### Running tcpdump / windump for the SUT` is probably converted to `--` by the github headline anchor generator.